src/main/java/bio/terra/cli/command/config/set/Browser.java

```diff
@@ -1,38 +1,30 @@
 package bio.terra.cli.command.config.set;
 
 import bio.terra.cli.auth.AuthenticationManager.BrowserLaunchOption;
-import bio.terra.cli.context.GlobalContext;
-import java.util.concurrent.Callable;
+import bio.terra.cli.command.helperclasses.BaseCommand;
 import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the fourth-level "terra config set browser" command. */
 @Command(
     name = "browser",
     description = "Configure whether a browser is launched automatically during the login process.")
-public class Browser implements Callable<Integer> {
+public class Browser extends BaseCommand {
 
   @CommandLine.Parameters(
       index = "0",
       description = "Browser launch mode: ${COMPLETION-CANDIDATES}")
   private BrowserLaunchOption mode;
 
+  /** Updates the browser launch option property of the global context. */
   @Override
-  public Integer call() {
-    GlobalContext globalContext = GlobalContext.readFromFile();
-
-    BrowserLaunchOption prevBrowserLaunchOption = globalContext.browserLaunchOption;
+  protected void execute() {
     globalContext.updateBrowserLaunchFlag(mode);
+  }
 
-    System.out.println(
-        "Browser launch mode for login is "
-            + globalContext.browserLaunchOption
-            + " ("
-            + (globalContext.browserLaunchOption == prevBrowserLaunchOption
-                ? "UNCHANGED"
-                : "CHANGED")
-            + ").");
```

Comment:
just curious, no more output? Some feedback to the user seems reasonable. Ditto on other classes.

Reply:
You know, I had just convinced myself that this was not really useful. Maybe I've been staring at it too long. Happy to put it back if you think it's helpful.